install/installing-knative-serving.adoc

```diff
@@ -27,4 +27,6 @@ include::modules/serverless-install-serving-yaml.adoc[leveloffset=+1]
 [id="next-steps_installing-knative-serving"]
 == Next steps
 
-* If you want to use Knative event-driven architecture you can xref:../install/installing-knative-eventing.adoc#installing-knative-eventing[install Knative Eventing].
+* After installing Knative Serving, you can start xref:../knative-serving/getting-started/serverless-applications.adoc#serverless-applications[creating serverless applications].
```

Comment:
Same consistency comments that align with [this guidance](https://github.com/openshift/openshift-docs/blob/main/contributing_to_docs/doc_guidelines.adoc#lead-in-link-sentences).

Reply:
Thank you for the reference!